helm_lib/templates/_module_security_context.tpl

```diff
@@ -98,3 +98,17 @@ securityContext:
     - ALL
     add: {{ index . 1 | toJson }}
 {{- end }}
+
+{{- /* Usage: {{ include "helm_lib_module_container_security_context_capabilities_drop_all_and_run_as_user_custom" (list . 1000 1000) }} */ -}}
+{{- /* returns SecurityContext parameters for Container with read only root filesystem, all dropped, and custom user ID */ -}}
+{{- define "helm_lib_module_container_security_context_capabilities_drop_all_and_run_as_user_custom" }}
```

Comment:
@name212 please pay attention here, to keep it synchronized with the separated helm_lib